Makefile

```diff
@@ -59,14 +59,11 @@ kotsadm:
 # Debugging
 .PHONY: kotsadm-debug-build
 kotsadm-debug-build:
-	go build ${LDFLAGS} -gcflags="all=-N -l" \
- 		$(BUILDFLAGS) \
-		-o ./bin/kotsadm-api-debug \
-		./cmd/kotsadm
+	CGO_ENABLED=1 go build -v -o ./bin/kotsadm-api-debug ./cmd/kotsadm
 
 .PHONY: kotsadm-debug
 kotsadm-debug: kotsadm-debug-build
-	dlv --listen=:2345 --headless=true --api-version=2 exec -- ./bin/kotsadm-api-debug
+	LOG_LEVEL=$(LOG_LEVEL) dlv --listen=:2345 --headless=true --api-version=2  exec ./bin/kotsadm-api-debug api
```

Comment:
as far as I know this LOG_LEVEL environment variable is unused.

Reply:
It is used in 
https://github.com/replicatedhq/kots/blob/8985152ebb9c40cc64ff53c9e1a85f66d63af05b/cmd/kotsadm/cli/api.go#L25-L27